manage_breast_screening/mammograms/jinja2/mammograms/components/card_with_status.jinja

```diff
@@ -0,0 +1,32 @@
+{% from "nhsuk/components/card/macro.jinja" import card %}
+{% from "nhsuk/components/tag/macro.jinja" import tag %}
+
+{% macro card_with_status(section, anchor, status_tag_properties, action_button_properties=None, csrf_input=None) %}
+  {% set outer_caller = caller if caller is defined else none %}
+  <div id="{{ anchor }}" class="app-reviewable-section">
+    {% call card({
```

Comment:
I'm assuming you won't need to change anything here, but just so you're aware there are some changes coming soon to the card and summary list components: https://github.com/nhsuk/nhsuk-frontend/pull/1685

For the medical history cards (nested within this one) I'm planning to make use of the new actions in the top right of the card.